src/main/java/reposense/report/RepoCloner.java

```diff
@@ -94,29 +92,25 @@ public void cleanup() {
     }
 
     /**
-     * Spawns a process to clone repo specified in {@code repoConfig}. Does not wait for process to finish executing.
+     * Spawns a process to clone repo specified in {@code config}. Does not wait for process to finish executing.
      * Should only handle a maximum of one spawned process at any time.
      */
     private boolean spawnCloneProcess(String outputPath, RepoConfiguration config) throws IOException {
         assert(crp == null);
 
         try {
-            GitLsTree.validateFilePaths(config);
-
-            FileUtil.deleteDirectory(config.getRepoRoot());
-            logger.info(String.format(MESSAGE_START_CLONING, config.getLocation()));
             Path rootPath = Paths.get(FileUtil.REPOS_ADDRESS, config.getRepoFolderName());
             Files.createDirectories(rootPath);
-            crp = runCommandAsync(rootPath, "git clone " + addQuote(config.getLocation().toString()));
+            FileUtil.deleteDirectory(FileUtil.getBareRepoPath(config).toString());
+            Files.createDirectories(Paths.get(FileUtil.REPOS_ADDRESS, config.getRepoFolderName()));
```

Comment:
![image](https://user-images.githubusercontent.com/35621726/60866223-62ceff00-a25a-11e9-8179-42bceaf071c9.png)
Seems to create the same directory for two times?

Reply:
My bad, overlooked this